### DISCUSSION TITLE
Align div in PDF  impossible

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#16463] [How to]
May I have advice on how to align following div to the right? none padding or margin working.

`<div style="padding-left:50%">
<div style="background-color:#dddddd; display:inline-block; width:20%">
<h1>Block1</h1>
</div>

<div style="background-color:#ffffff; display:inline-block; text-align:center; width:10%">space</div>

<div style="background-color:#dddddd; display:inline-block; width:20%">
<h1>Block2</h1>
</div>
</div>
`

**Comment by s2b-git** [comment]
use css block-inline and margin
